resim/time/BUILD

```diff
@@ -10,7 +10,8 @@ cc_library(
     name = "timestamp",
     hdrs = ["timestamp.hh"],
     visibility = ["//visibility:public"],
-    deps = [],
+    deps = [
```

Comment:
?

Reply:
I added some deps while debugging and trunk fmt doesn't appear to revert this back to a single line. 🤔